draft-ietf-mls-protocol.md

```diff
@@ -4946,16 +4922,10 @@ To construct the tree in {{evolution-tree}}:
 
 To construct the tree in {{parent-hash-tree}}:
 
-* A creates a group with B, ... G
+* A creates a group with B, ..., G
 * A removes F in a full Commit, setting T, U, and W
-* E sends an empty Commit, setting X, Y, and W
-* A adds a new member at F in a partial Commit, adding F as unmerged at X, Y,
-  and W
-* A removes D, resetting T, U, and W (in particular, F is no longer unmerged at
-  W)
-* A adds new memebers at D and H in a partial commit
-  * D is added as unmerged at U, W
-  * H is added as unmerged at Y, W
+* E sends an empty Commit, setting Y and W
+* A adds a new member at F in a partial Commit, adding F as unmerged at Y and W
```

Comment:
```suggestion
* A adds a new member at F in a partial Commit, adding F as unmerged at X, Y, and W
```

Reply:
I removed X from the tree, so the current version is correct again.